pom.xml

```diff
@@ -114,17 +115,14 @@
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter</artifactId>
-            <version>${spring.boot.version}</version>
```

Comment:
I removed the version for a particular dependencies to avoid conflicts with other dependencies. Specifically, there was a issue of versions collision between `pbs.jar` and `pbs-bundle.jar`.